src/cpp/meson.build

```diff
@@ -12,6 +12,7 @@ else
     extra_cpp_args = []
 endif
 
+boost_dep = dependency('boost', version : '>=1.56' ) # we only use headers : boost::random is templates-only
```

Comment:
we need to finally finish meson or just switch to a pure python package with a CPython library.so

Reply:
Due to some pressing targets (needing ShoRAH 2 for processing COVID-19) I would suggest we post-pone the meson-ification of the build system to a later point in the 2.x version cycle.

Would you agree?

Reply:
I didnt say you have to do it now, just at some point in the future.